src/app/shared/pipes/duration.pipe.ts

```diff
@@ -1,8 +1,18 @@
-import { Pipe } from "@angular/core";
+import { Pipe, PipeTransform } from "@angular/core";
 
 @Pipe({
     name: 'duration'
 })
-export class DurationPipe {
-    // Add your code here
+export class DurationPipe implements PipeTransform{
+
+    transform(durationInMinutes: number): string {
```

Comment:
You can handle 'unwanted' values that is coming form the user input here.

Reply:
Done according to previous comment